internal/services/controllers/jobs/controller.go

```diff
@@ -553,8 +555,13 @@ func (ec *JobsControllerImpl) handleStepRunQueued(ctx context.Context, task *msg
 	return ec.queueStepRun(ctx, metadata.TenantId, metadata.StepId, payload.StepRunId)
 }
 
-func (jc *JobsControllerImpl) runStepRunRequeue(ctx context.Context) func() {
+func (jc *JobsControllerImpl) runStepRunRequeue(ctx context.Context, startedAt *time.Time) func() {
```

Comment:
any reason this is a pointer when the single execution always provides a value?

Reply:
because we're saving TONs of memory :)

... there's no real reason, I usually default to passing time.Time as a pointer

Reply:
not a big deal but it seems like if the value is required it could make more sense to use a value, especially as you explicitly need to check for != nil.